### PR TITLE
Added questions in Check for Understanding

### PR DESCRIPTION
**Pull request by raibove**
Added questions in Check for Understanding for topic javascript-1

Issue : Techtonica#700

### CODE REVIEW

**Comment by raibove** [comment]
@Kundan28  @saberprashant  can you please help me to generate this PR. My previous issue are also getting added in this as a commit.

**Comment by imskr** [comment]
Please be sure to fetch the base branch before pushing. It is creating conflict because your branch is older.

File: javascript/javascript-1.md [hunk 1]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
 
 ### Check for Understanding
 
+ Form small groups and discuss:
+ 
+- Mention some rules for declaring variables in JavaScript?
+- Mention some math operations that can be performed on number variables in JavaScript?
+- How to define and call a function in JavaScript?
```
Comment (alodahl) [inline javascript/javascript-1.md]: "How *do you* define"...

File: javascript/javascript-1.md [hunk 2]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
 
 ### Check for Understanding
 
+ Form small groups and discuss:
+ 
+- Mention some rules for declaring variables in JavaScript?
```
Comment (alodahl) [inline javascript/javascript-1.md]: *What are* some rules for...

File: javascript/javascript-1.md [hunk 3]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
...
+- Mention some math operations that can be performed on number variables in JavaScript?
```
Comment (alodahl) [inline javascript/javascript-1.md]: *What are* some math...

File: javascript/javascript-1.md [hunk 4]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
...
+- How to define and call a function in JavaScript?
+- How to find the length of a string?
```
Comment (alodahl) [inline javascript/javascript-1.md]: How *do you* find...

File: javascript/javascript-1.md [hunk 5]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
...
+- Index of a string in JavaScript begins from?
```
Comment (alodahl) [inline javascript/javascript-1.md]: Are you asking about the first index being 0, @raibove ?

You can say:
"What is the first index of a string?"

File: javascript/javascript-1.md [hunk 6]
```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
 
 ### Check for Understanding
 
+ Form small groups and discuss:
```
Comment (alodahl) [inline javascript/javascript-1.md]: These are good questions overall. There are some grammar changes needed.

File: javascript/javascript-1.md (hunk 4 above)
Comment (alodahl) [reply javascript/javascript-1.md]: and *How do you* define (line 113)

**Comment by alodahl** [comment]
Mentors, can you make sure that these questions address the "objectives" of the lesson?

File: roles-in-tech/product-manager.md [hunk 7]
```diff
@@ -1,102 +0,0 @@
-# Roles in Tech: Product Manager
```
[the remark this answers is not in the document]
Comment (raibove) [reply roles-in-tech/product-manager.md]: @alodahl  Sorry it was a mistake. I have updated the file product-manager.md

**Comment by raibove** [comment]
@alodahl please review the changes

**Comment by raibove** [comment]
I had copied product manager file from original file itself.

**Pull request merged**